pennylane/qaoa/mixers.py

```diff
@@ -112,3 +114,85 @@ def xy_mixer(graph):
         obs.append(qml.PauliY(node1) @ qml.PauliY(node2))
 
     return qml.Hamiltonian(coeffs, obs)
+
+
+def bit_flip_mixer(graph, b):
+    r"""Creates a bit-flip mixer Hamiltonian.
+
+    This mixer is defined as:
+
+    .. math:: H_M \ = \ \displaystyle\sum_{v \in V(G)} \frac{1}{2^{d(v)}} X_{v}
+              \displaystyle\prod_{w \in N(v)} (\mathbb{I} \ + \ (-1)^b Z_w)
+
+    where :math:`V(G)` is the set of vertices of some graph :math:`G`, :math:`d(v)` is the
+    `degree <https://en.wikipedia.org/wiki/Degree_(graph_theory)>`__ of vertex :math:`v`, and
+    :math:`N(v)` is the `neighbourhood <https://en.wikipedia.org/wiki/Neighbourhood_(graph_theory)>`__
+    of vertex :math:`v`. In addition, :math:`\mathbb{I}` is the identity operator, and :math:`X_v` and :math:`Z_v`
```

Comment:
```suggestion
    of vertex :math:`v`. In addition, :math:`\mathbb{I}` is the identity operator, and :math:`X_v`, :math:`Z_v`
```
to avoid too many "ands"